### PR TITLE
Add documentation for constants.py

### PR DESCRIPTION
**Pull request by Kolloom**

### CODE REVIEW

File: docs/source/constants.rst [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+.. note::
+    ``MOBJECT_DIR`` and ``IMAGE_MOBJECT_DIR`` is deprecated.
+
+Tex
+---
+    TEX_USE_CTEX
+              A boolean value. Change it to True if you need to use Chinese typesetting.
+    TEX_TEXT_TO_REPLACE
+              ????????
```
Comment (eulertour) [inline docs/source/constants.rst]: This is the placeholder text in the tex template that Manim will replace when writing files[[1](https://github.com/3b1b/manim/blob/c42ae8a55bb2f1ef8259a719f17eaf519c46f3f8/manimlib/tex_template.tex#L24)][[2](https://github.com/3b1b/manim/blob/9b2c2c4784a9d619fed0b02d4af4555f12c1741c/manimlib/utils/tex_file_writing.py#L32-L34)].

File: docs/source/constants.rst [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+Camera Configuration
+--------------------
+
+Render setting presets
+
+    PRODUCTION_QUALITY_CAMERA_CONFIG
+            2560x1440 @ 60fps
+    HIGH_QUALITY_CAMERA_CONFIG
+            1920x1080 @ 60fps. This is the default when rendering a scene
```
Comment (eulertour) [inline docs/source/constants.rst]: The default is actually` PRODUCTION_QUALITY_CAMERA_CONFIG`[[1](https://github.com/3b1b/manim/blob/c42ae8a55bb2f1ef8259a719f17eaf519c46f3f8/manimlib/config.py#L220)].

File: docs/source/constants.rst [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    MEDIUM_QUALITY_CAMERA_CONFIG
+            1280x720 @ 30fps
```
Comment (eulertour) [inline docs/source/constants.rst]: You could mention that this is used when the `-m` flag is passed.

**Comment by Kolloom** [comment]
That should do it.

**Pull request merged**